bpfd-api/src/config.rs

```diff
@@ -136,7 +136,7 @@ fn default_unix() -> String {
 }
 
 pub fn config_from_file<P: AsRef<Path>>(path: P) -> Config {
-    if let Ok(contents) = fs::read_to_string(path) {
+    if let Ok(contents) = read_to_string(path) {
```

Comment:
Oh man you totally got me here. Turns out that we had some skeletons hidden in the closet here 💀 
I've pushed more changes now to fix this but TL;DR:

1. That we have duplicate config.rs in bpfd and bpfd-api where only the
       latter is used
1. That bpfd can be a binary only crate since everything exposed lives in `bpfd_api`
1. That we need to remove fs::read_to_string from bpf-api since bpfctl and bpfd might want to use different versions of read_to_string

So I ended up pushing that all in this PR 😓